.github/CONTRIBUTING.md

```diff
@@ -34,54 +35,50 @@ Unsure where to begin contributing to Chomp Station? You can start by looking th
+* Our random overmap POI files are in [modular_chomp/maps/overmap/].
 * Our surface POI map files (Wilderness, plains, ...) are in [modular_chomp/maps/submaps/surface_submaps/].
-* Map changes must be in tgm format. See the [Mapmerge2 Readme] for details, or use [StrongDMM] which can automatically save maps as tgm.
+* Map changes must be in tgm format. See the [Mapmerge2 Readme](../tools/mapmerge2/readme.md) for details, or use [StrongDMM](../tools/StrongDMM/README.md) which can automatically save maps as tgm.
+* PoIs or map templates placed during generation (Mining, plains, wilderness, space...) are generally fair game for editing or creating anew.
+* Maps that are placed permanently (Station, planetary locations) must be subject to discussion. To prevent wasted time, discuss these with the community and staff
+    * There is a limited budget of RAM available for permanent maps. This must be carefully rationed. 
+* Entire new station designs MUST be discussed with the community and the staff. Post a floor plan or basic design before committing effort in discord to prevent wasted time. 
+* For new atmospherics settings, subtypes of turfs for all turfs can be created with the [turfpacks](../modular_chomp/maps/~turfpacks/turfpacks.dm) system quickly and easily.
+
+## Chomp Station Coding Standards
+
+### General
+* **DO NOT** create joke or meme PRs. The Github is intended to be a sterile location for reviewing technical content.
+* We **DO NOT** allow any kind of CKEY/personally locked content on this codebase. Anything created must be available to all or none.
+    * Our upstream does allow this, and ckey-locked content from them does exist in our code. We try to comment it out where possible. Please report anything missing.
+    * If you have ckey locked content from our upstream and would like to make it available to all here, please contact us.
+* we **DO NOT** allow any 'naming' in our coded content. this includes shoutouts, naming a player as an owner or otherwise. All descriptions, names, lore-texts must be free of an individual's name. NPC naming is permitted.
+
+### Codewriting
+* Where possible, add changes to the modular_chomp/ subdirectory.
+* For edits to upstream code (Code not inside /modular_chomp)
+    * For single line edits, add "//ChompEDIT - description" at the end of the line.
+    * For block edits, encapsulate your edit with "//ChompEDIT START - explanation" and "//ChompEDIT END".
+    * For removals, comment out the upstream code and append "//ChompEDIT REMOVE - explanation"
+    * For block removals, use "/\*" and "\*/" with "//ChompEDIT REMOVE - explanations"
+* **DO NOT** edit upstream .dmi (icon) files. Add icon changes in a new file in the /modular_chomp folder and override the object's "icon" and "icon_state" variables. 
+* **DO NOT** edit anything in the upstream /maps folder 
+* Where possible and applicable, send bugfixes to Virgo (Our upstream) to fix at the source. this is not enforced, however.
+
```

Comment:
* usr should be avoided where possible. Use src or hand the reference to the user through the proc chain.
* old <span> tags are not to be used. Use the span_type defines.